### ISSUE TITLE
Remove GMaps

### ISSUE THREAD

**Issue by Matt-Jensen** [Matt-Jensen/ember-cli-g-maps#66]
No one wants an Addon that requires bower dependencies and GMaps manages map children, which just isn't disco.  Goals are to remove `gmaps-for-apps.js` and manage Google Map children as regular components.

- [ ] Google Map component (and factory module if additional abstraction is necessary)
- [ ] Google Marker component
- [ ] Google Circle component
- [ ] Google Polygon component
- [ ] Google Polyline component
- [ ] Google Rectangle component
- [ ] Google Overlay component
- [ ] Separate version 1 documentation
- [ ] Remove GMaps app.import and update `bower.json`

**Comment by tsteuwer** [comment]
Need any help with this?

**Comment by Matt-Jensen** [comment]
@tsteuwer Absolutely.  This branch has a lot going on so I'm more than happy to do a walk through with you. However if you're feeling confident I haven't started Overlay Views and Info Windows.

**Comment by devotox** [comment]
Ha i saw your message only after making the PR but it does seem to work with your current gmap for apps github package
